olive/passes/onnx/vitis_ai/refine.py

```diff
@@ -8,10 +8,9 @@
 
 from olive.passes.onnx.vitis_ai.quant_utils import pos2scale, scale2pos
 
-logger = logging.getLogger(__name__)
-
 refine_op_type = ["DequantizeLinear", "QuantizeLinear"]
 postfix = "_Output"
+logging.basicConfig(level=logging.INFO)
```

Comment:
can you please use the `logger = logging.getLogger(__name__)` along with `logger.info`, `logger.debug`? This makes the logs from all olive code under the `olive` namespace. 

`logging.basicConfig(level=logging.INFO)` is not recommended for python libraries since it changes the root logger. The root logger should only be changed by the user themselves.

Reply:
Jambay is right, there should only one global logging.basicConfig call in one python process.